ldk/javascript/src/webpack/generate-ldk-settings.ts

```diff
@@ -0,0 +1,34 @@
+import * as path from 'path';
+import { customizeArray, CustomizeRule, mergeWithCustomize } from 'webpack-merge';
+import { LdkSettings } from './ldk-settings';
+
+export function mergeLdkSettings(
+  baseSettings: LdkSettings,
+  overrideSettings: LdkSettings,
+): LdkSettings {
+  return mergeWithCustomize<LdkSettings>({
+    customizeArray: customizeArray({
+      'ldk.permissions.network.urlDomains.*': CustomizeRule.Replace,
+      'ldk.permissions.filesystem.pathGlobs.*': CustomizeRule.Replace,
+    }),
+  })(baseSettings, overrideSettings);
+}
+
+export function generateLdkSettings(): LdkSettings {
+  // Need to dynamically refer to Loop's package.json
+  // Suppressing rule as we intentionally want a dynamic require.
+  // eslint-disable-next-line @typescript-eslint/no-var-requires,import/no-dynamic-require,global-require
+  let ldkSettings: LdkSettings = require(path.join(process.cwd(), '/package.json'));
+
+  if (process.env.LDK_CONFIG !== undefined) {
```

Comment:
Just thought of something on this part - I'm not sure what happens if you request a config and there's no appropriate file present. This probably needs to be in a try/catch - will test out and update this.

Reply:
You mean no `/package.json`? You could test for that but I feel as if you do not include that you don't have a good chance at generating a transpiled Loop anyway. I do think you'll have to check for no `/ldk-config.*` before you require though as that is optional.

Reply:
Yeah, I mean no `ldk-config`. The main case is probably if you mistype the config name (e.g. `LDK_CONFIG=devv npm run build` or similar). 

It throws a big nasty error if that happens, which is probably the right behavior. I'll add some extra logging though in the `catch`. Coming shortly.

Reply:
Added logging - resolving this.